### PR TITLE
Various fastlane updates

### PR DESCRIPTION
**Pull request by yleflour**

### CODE REVIEW

File: generators/fastlane-env/index.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class FastlaneEnvGenerator extends Base {
     );
     this.fs.copyTpl(
       this.templatePath('environment/index.js'),
-      this.destinationPath(`environment/index.${this.answers.environmentName}.js`),
+      this.destinationPath(`src/environment/index.${this.answers.environmentName}.js`),
```
Comment (Almouro) [inline generators/fastlane-env/index.js]: 👍

File: generators/fastlane-setup/templates/env [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Build number is composed of :
```
Comment (Almouro) [inline generators/fastlane-setup/templates/env]: Why grouping this in a single file? 
What if you have to manage different apps with different versions? (example: Storyssimo + Instant Parisien)

File: generators/fastlane-setup/templates/fastlane/Fastfile [hunk 3]
```diff
@@ -52,10 +52,12 @@ platform :ios do
     match(
       type: 'development',
       shallow_clone: true,
+      clone_branch_directly: true,
```
Comment (Almouro) [inline generators/fastlane-setup/templates/fastlane/Fastfile]: 👍

File: generators/fastlane-setup/templates/env (hunk 2 above)
Comment (yleflour) [reply generators/fastlane-setup/templates/env]: Then you split it up

**Pull request merged**